networking/networking.go

```diff
@@ -229,18 +233,28 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
+func (n *Networking) GetDefaultIP() (net.IP, error) {
```

Comment:
I suggest documenting this method and its behaviour returning the runtime IP of the first masqueraded network or an err if none was found.